addons/xeh/fnc_startFallbackLoop.sqf

```diff
@@ -28,8 +28,7 @@ GVAR(fallbackRunning) = true;
 {
     // don't run init and initPost event handlers on objects that already exist
     SETINITIALIZED(_x);
```

Comment:
SETINITIALIZED is a setVariable which returns nil, which would make the count work here.

Reply:
As this is something that runs every frame and the array can be big in certain missions, I agree that it can use count loop here.